AirCasting/Utils/Strings.swift

```diff
@@ -4,6 +4,14 @@
 import Foundation
 
 struct Strings {
+    enum Commons {
+        static let cancel: String = "Cancel"
+        static let ok: String = "OK"
+        static let myAccount: String = "My account"
+        static let `continue`: String = "Continue"
```

Comment:
why are there those `` around continue?

Reply:
Thank you for mentioning what I have left. This `` is used to make continue a let as the others because normally continue is a reserved keyword in Swift (Special thanks to @pawelgil for that knowledge!)